book/src/using-bitfields.md

```diff
@@ -0,0 +1,196 @@
+# Using the Bitfield Types Generated by Bindgen
+
+For this discussion, we will use the following C type definitions and functions.
+```c
+typedef struct {
+    unsigned int a: 1;
+    unsigned int b: 1;
+    unsigned int c: 2;
+    
+} bitfield;
+
+// Create a default bitfield
+bitfield create_bitfield();
+
+// Print a bitfield
+void print_bitfield(bitfield bfield)
```

Comment:
Nitpick: missing a semicolon at the end of this declaration.

Reply:
Done